src/CalcManager/UnitConverter.cpp

```diff
@@ -649,7 +656,7 @@ vector<tuple<wstring, Unit>> UnitConverter::CalculateSuggested()
         if (stod(roundedString) != 0.0 || m_currentCategory.supportsNegative)
         {
             TrimTrailingZeros(roundedString);
-            returnVector.push_back(make_tuple(roundedString, entry.type));
+            returnVector.emplace_back(roundedString, entry.type);
```

Comment:
Using `emplace_back` here instead of `push_back` so that the tuple can be directly constructed in the container vs. constructed and moved into the container.